mapie/tests/test_classification.py

```diff
@@ -1367,6 +1367,8 @@ def test_valid_prediction(alpha: Any) -> None:
 @pytest.mark.parametrize("strategy", [*STRATEGIES])
 def test_toy_dataset_predictions(strategy: str) -> None:
     """Test prediction sets estimated by MapieClassifier on a toy dataset"""
+    if strategy == "aps_randomized_cv_crossval":
```

Comment:
How come this test fails? Or why did we add this?

Reply:
Could you put the link of the new issue?

Reply:
https://github.com/scikit-learn-contrib/MAPIE/issues/377